app/assets/javascripts/qsets.js

```diff
@@ -27,27 +30,34 @@ function initQsets() {
   $('form').on('submit', function (event) {
     $('.modal :button').prop('disabled', true);
   });
+
+  $('.quiz-all').on('click', function (event) {
```

Comment:
I think you can use [the html attribute way to show modals](http://getbootstrap.com/javascript/#live-demo), like we did for [the old quiz buttons](https://github.com/penzance/askup/pull/85/files#diff-e7a9d9091a735cfbbfd581f046d52edcL60) (see the `data-toggle` and `data-target` attributes), and do away with this JS completely. This is a matter of preference, I'd say, so your choice, but personally I think it's easier to tell what the button does if you use the html attribute method -- otherwise you'd not have any idea what the effects were by reading the html template.